app-frontend/src/app/components/sceneList/sceneList.html

```diff
@@ -0,0 +1,50 @@
+<div class="list-group" ng-show="$ctrl.loading">
+  <span class="list-placeholder">
+    <i class="icon-load"></i>
+  </span>
+</div>
+<div class="list-group">
+  <rf-scene-item
+      on-view="$ctrl.viewSceneDetail(scene)"
+      on-download="$ctrl.downloadModal(scene)"
+      scene="scene"
+      editable
+      ng-repeat="scene in $ctrl.sceneList track by scene.id">
+  </rf-scene-item>
+</div>
+<div class="list-group text-center"
+     ng-show="!$ctrl.loading && $ctrl.lastSceneResult && $ctrl.lastSceneResult.count > $ctrl.lastSceneResult.pageSize && !$ctrl.errorMsg">
```

Comment:
Definitely agree -- I had thought about doing that when I pulled this in from the Scenes detail page but it slipped my mind. Will update.